src/DIRAC/WorkloadManagementSystem/Client/PilotLoggingPlugins/FileCacheLoggingPlugin.py

```diff
@@ -0,0 +1,133 @@
+"""
+File cache logging plugin.
+"""
+import os
+import json
+import re
+from DIRAC import S_OK, S_ERROR, gLogger
+
+sLog = gLogger.getSubLogger(__name__)
+
+
+class FileCacheLoggingPlugin:
```

Comment:
Shouldn't this inherit from the base?

```suggestion
class FileCacheLoggingPlugin(PilotLoggingPlugin):
```

Reply:
Done.